fixup-google-java-format.py

```
@@ -360,9 +355,9 @@ def eprint(*args: object, **kwargs: Any) -> None:
 }
 
 # File .type-annotations can add to the type_annotations variable.
-if pathlib.Path(".type-annotations").is_file():
-    type_annotations_text = pathlib.Path(".type-annotations").read_text()
-    exec(type_annotations_text)
+_type_annotations_path = pathlib.Path(".type-annotations")
+if _type_annotations_path.is_file():
+    exec(_type_annotations_path.read_text())
```

Ruff (0.15.9)

[error] Use of `exec` detected

(S102)